### ISSUE TITLE
BentoML Integration with the Flux Operator

### ISSUE THREAD

**Issue by vsoch** [bentoml/BentoML#3836]
hi! I'm a developer for the [Flux Operator](https://github.com/flux-framework/flux-operator), which (from a high level) let's you deploy an HPC (supercomputer) cluster on Kubernetes. It comes down to a set of networked pods (via an indexed job) and we've been able to run distributed setups on it from Tensorflow, Pytorch, etc.

I just stumbled on BentoML, and aside from loving the idea and docs (I love the dark background with pink!) I'm interested in understanding how I might deploy Bento in our operator. What I can easily do now is launch (and expose) a service from a main pod that we call the broker (that has created a network for the other pods) and I could easily run an equivalent bento start command in the same way. What I'm interested in is how to setup or configure bento so that it can register / be aware of other machines (akin to some of the distributed ML libraries to provide hostnames) _or_ I could write a plugin that would work directly with the flux scheduler to allocate / get results back for tasks from the main server.

Apologies for my ignorance about how everything works - I just stumbled on it this morning and am really excited to try it out! If you have suggestions for the above, what I might look at first, it would be greatly appreciated!

**Comment by vsoch** [comment]
What exactly about this issue is "not planned" - a person actually responding to me? In a timeframe of years later? 🙃
